bug_report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug Report
+about: This template is for reporting bugs you have found.
+title: "[DATE]: [BUG TITLE]"
+labels: Needs Triage
+assignees: 
+---
+
+## Information
+
+Device:
```

Comment:
Perhaps remove this part ( I think we should make the original files a bit more generic :( )